spec/paperclip/content_type_detector_spec.rb

```diff
@@ -18,6 +18,13 @@
     assert_equal "video/mp4", Paperclip::ContentTypeDetector.new(@filename).detect
   end
 
+  it 'returns content type of file if it is an acceptable type' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.